drivers/include/periph/pwm.h

```diff
@@ -18,84 +18,91 @@
  * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
  */
 
-#ifndef PWM_H
-#define PWM_H
+#ifndef PERIPH_PWM_H
+#define PERIPH_PWM_H
 
-#include "periph_conf.h"
+#include <stdint.h>
+
+#include "periph_cpu.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-/* ignore file in case no PWM devices are defined */
-#if PWM_NUMOF
-
 /**
- * @brief Definition of available PWM devices
- *
- * To this point a maximum of four PWM device is available.
+ * @brief   Default PWM access macro
+ * @{
  */
-typedef enum {
-#if PWM_0_EN
-    PWM_0,              /*< 1st PWM device */
-#endif
-#if PWM_1_EN
-    PWM_1,              /*< 2nd PWM device */
+#ifndef PWM_DEV
+#define PWM_DEV(x)          (x)
```

Comment:
It is not used (yet). This define provides a universal means of accessing the different PWM devices in a more programmable fashion, while still providing the possibility for each CPU to override this default behavior. Have a look at the `GPIO` define and it's usage in `drivers/include/periph/gpio.h` and how some CPUs use it (e.g. `cpu/stm32f4/include/periph_cpu.h`)...